pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -87,6 +87,8 @@ func NewMustGatherCommand(f kcmdutil.Factory, streams genericclioptions.IOStream
 	}
 
 	cmd.Flags().StringVar(&o.NodeName, "node-name", o.NodeName, "Set a specific node to use - by default a random master will be used")
+	cmd.Flags().StringVar(&o.NodeSelector, "node-selector", o.NodeSelector, "Set a specific node selector to use - only relevant when specifying a specific command and image which needs to capture data on a set of cluster nodes simultaneously")
+	cmd.Flags().BoolVar(&o.HostNetwork, "host", o.HostNetwork, "Run must-gather pods as hostNetwork: true - relevant if a specific command and image needs to capture host-level data")
```

Comment:
`host-network` let's make the flag explicit

Reply:
Done

Reply:
done